tsconfig.json

```diff
@@ -7,7 +7,6 @@
     "sourceMap": true,
     "target": "es2017",
     "lib": ["es2017","es2019.array", "es2018.promise", "dom", "dom.iterable"],
-    "types": ["web-ext-types"],
```

Comment:
Maybe this should be an empty array? The error makes it sound like jest failed to translate the test files from ts to js, which to me means that there's probably an issue with the ts config.

Reply:
No luck - that breaks the build (Typescript automatically uses all `@types` folders unless you specify anything there) and doesn't fix the test either.

Logically, manually specifying `@types/firefox-webext-browser` doesn't work either.